app/models/event.rb

```diff
@@ -194,16 +175,68 @@ def ticket_requests_open?
     false
   end
 
-  def eald?
-    early_arrival_price.positive? || late_departure_price.positive?
-  end
-
   def to_param
     return nil unless persisted?
 
     [id, slug].join('-') # 1-summer-campout-xii
   end
 
+  def build_event_addons_from_params(build_params)
```

Comment:
Perhaps use before_validation to create default values.

Reply:
[https://github.com/fnf-org/TicketBooth/issues/250](url) Opened issue for this to resolve later.